htdocs/accountancy/customer/list.php

```diff
@@ -235,6 +235,7 @@
 	$sql .= " AND f.type IN (" . Facture::TYPE_STANDARD . "," . Facture::TYPE_STANDARD . "," . Facture::TYPE_CREDIT_NOTE . "," . Facture::TYPE_DEPOSIT . "," . Facture::TYPE_SITUATION . ")";
 }
 $sql .= " AND f.entity IN (" . getEntity('facture', 0) . ")";    // We don't share object for accountancy
+$sql .= " AND aa.entity = " . $conf->entity;
```

Comment:
Because there is a left join, i think the AND should be added on line 201 instead of here.